src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputEnumTypeConverter.cs

```diff
@@ -148,17 +150,5 @@ private static IReadOnlyList<InputEnumTypeValue> NormalizeValues(IReadOnlyList<I
 
             return concreteValues;
         }
-
-        //private static IReadOnlyList<InputEnumTypeValue> convertToConcreteValues<EnumType, ValueType>(IReadOnlyList<InputEnumTypeValue> allowedValues) where EnumType : InputEnumTypeValue
```

Comment:
should this be removed?

Reply:
I saw it is commented and never used, I just removed it.
Do we still need it in the future? I could revert this removal